ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/external-ccm/cucushift-installer-rehearse-vsphere-upi-external-ccm-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/vsphere/upi/external-ccm/cucushift-installer-rehearse-vsphere-upi-external-ccm-workflow.yaml",
```

Comment:
I suggest to replace `external-ccm` with `ccm/external`, so that we can easily place more install workflow if we want to cover more ccm type

Reply:
ok, thanks! Let me update it.